DependencyInjection/DoctrineExtension.php

```diff
@@ -62,19 +62,19 @@ public function load(array $configs, ContainerBuilder $container)
             $this->dbalLoad($config['dbal'], $container);
         }
 
-        if (empty($config['orm'])) {
-            return;
+        if (! empty($config['orm'])) {
+            $this->ormLoad($config['orm'], $container);
```

Comment:
this must happen after the `if (empty($config['dbal'])) {` and after the check that the ORM is installed. Otherwise, you miss the existing validation before loading the ORM stuff (validating after does not perform the same, and may not fail properly when the ORM is missing)

Reply:
@stof I've moved loading messenger service before ORM which installation check was moved inside ormLoad method. Also fixed dbal configuration check